.pre-commit-hooks.yaml

```diff
@@ -4,8 +4,16 @@
   language: docker_image
   entry: returntocorp/semgrep:develop
 
-- id: semgrep
+- id: semgrep-docker
   name: semgrep
   description: This hook runs semgrep
   language: docker_image
   entry: returntocorp/semgrep:latest
+
+- id: semgrep
```

Comment:
Will we be breaking backwards compatibility at all for existing pre-commit configurations? I don't _think_ so. Since the Docker image uses the Python package and they present the same interface I think we're fine here. Thoughts?

If there's no backwards compatibility issues, and we go with `semgrep-docker`, we should update the first one to `semgrep-docker-develop`. Although we'll also have to update a few of our internal pre-commit configurations. Maybe it's not worth it, what do you think?

Reply:
There should not be any issues since when you specify a pre-commit hook you specify the git tag the repo is checked out at so any existing users that use the id semgrep will still actually use the docker container one.